src/CustomButtons.js

```diff
@@ -0,0 +1,8 @@
+export function CustomButtons({handleClick}){
```

Comment:
add some spacing in the parameter section for better readability
```suggestion
export function CustomButtons({ handleClick }){
```

Reply:
Thanks for the review, and I wanted to let you know that I Have added space in the parameters.